_docs/pql_queries.md

```diff
@@ -20,7 +20,7 @@ The syntax isn't alway very intuitive. That's why this page exists. It serves as
 
 All queries here are documented in the [string-based query style](https://puppet.com/docs/puppetdb/7/api/query/v4/pql.html), not the [AST](https://puppet.com/docs/puppetdb/7/api/query/v4/ast.html) style. You need the [Puppet Client tools](https://puppet.com/docs/puppetdb/7/pdb_client_tools.html) (for Open Source or PE) to get the `puppet query` face. To install the client tools on Open Source, you can do:
 
-```
+```console
```

Comment:
This doesn't look like `console`, which is a command prefixed by either `#` (for root) or `$` (for non-root). Then unprefixed lines are the command output. If anything, this is `shell`.

Reply:
Yeah, it was a toss up since they aren't colorized either way, mostly just like to eliminate warnings from vscode. I'll go with `shell`.

Reply:
Ah actually `shell` does color the quoted parts. 👍